src/main/java/emu/grasscutter/game/player/Player.java

```diff
@@ -1202,24 +1207,11 @@ public void loadFromDatabase() {
     }
 
     public void onPlayerBorn() {
-        if (Grasscutter.getConfig().server.game.gameOptions.questing) getQuestManager().onPlayerBorn();
+        getQuestManager().onPlayerBorn();
+        // if (Grasscutter.getConfig().server.game.gameOptions.questing) getQuestManager().onPlayerBorn();
```

Comment:
This check should stay there, in case people don't want quests to be active in a sandbox.

Reply:
I removed it because it was blocking me from enabling quest from start, where should i get the questing option enebled?

Reply:
in the config.json und server->game->questing
if the config is older it might not be there yet, then just add it as boolean.